bin/check_tron_jobs.py

```diff
@@ -156,7 +156,7 @@ def pretty_print_actions(action_run):
 def get_relevant_run_and_state(job_content):
     job_runs = sorted(
         job_content.get('runs', []),
-        key=lambda k: k['run_time'],
+        key=lambda k: (k['end_time'] is None, k['end_time'], k['run_time']),
```

Comment:
I wrote out a few examples for myself, and it seems like this sort key translates into:
1) Currently running ones are always checked first (in case an action is failed/unknown)
2) If there are multiple running ones, then most recent run_time wins
3) If nothing is currently running, then most recent end_time wins

Is that correct? Can we add a comment about the intended behavior so we/others can remember in the future?

Reply:
Yes, it's right. I will make a comment in the code.